android/gradle.properties

```diff
@@ -1,3 +0,0 @@
-
-android.useAndroidX=true
-android.enableJetifier=true
```

Comment:
Is this our change or done by library?

Reply:
This was our change which was done earlier. This is not present In library code.

Reply:
Already discussed